cms/methodology/models.py

```diff
@@ -0,0 +1,91 @@
+from typing import TYPE_CHECKING, Any, ClassVar
+
+from django.conf import settings
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.utils.functional import cached_property
+from django.utils.translation import gettext_lazy as _
+from wagtail.admin.panels import FieldPanel, InlinePanel, MultiFieldPanel
+from wagtail.fields import RichTextField
+from wagtail.search import index
+
+from cms.core.blocks.stream_blocks import SectionStoryBlock
+from cms.core.fields import StreamField
+from cms.core.models import BasePage
+
+if TYPE_CHECKING:
+    from django.http import HttpRequest
+    from wagtail.admin.panels import Panel
+
+
+class MethodologyPage(BasePage):  # type: ignore[django-manager-missing]
+    parent_page_types: ClassVar[list[str]] = ["topics.TopicPage"]
+
+    template = "templates/pages/methodology_page.html"
+
+    summary = RichTextField(features=settings.RICH_TEXT_BASIC)
+    publication_date = models.DateField()
+    last_revised_date = models.DateField(blank=True, null=True)
+
+    contact_details = models.ForeignKey(
+        "core.ContactDetails",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
+
+    content = StreamField(SectionStoryBlock())
+
+    show_cite_this_page = models.BooleanField(default=True)
+
+    content_panels: ClassVar[list["Panel"]] = [
+        *BasePage.content_panels,
+        FieldPanel("summary"),
+        MultiFieldPanel(
+            [
+                FieldPanel("publication", icon="calendar-date"),
```

Comment:
```suggestion
                FieldPanel("publication_date", icon="calendar-date"),
```

Reply:
Fixed as part of [1ea7ed](https://github.com/ONSdigital/dis-wagtail/pull/57/commits/1ea7ed1e96e994acd51f46a43ee29a3956aa6c39)